app/controllers/users_controller.rb

```diff
@@ -1,7 +1,50 @@
 class UsersController < ApplicationController
 
-  def list
+  def index
     @users = User.all
   end
-  alias :index :list
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    respond_to do |format|
```

Comment:
You don't need to use ```respond_to``` unless you are actually going to handle other types of formats like JSON, JS, etc.